formatTest/typeCheckedTests/expected_output/attributes.rei

```diff
@@ -29,8 +29,7 @@ external createClassInternalHack :
   "createClass";
 
 [@bs.send.pipe: array('a)]
-external map : [@bs] (('a => 'b) => array('b)) =
-  "";
+external map : (. ('a => 'b)) => array('b) = "";
```

Comment:
the dot position is a bit weird, should not it be 
```reason
external map : ((. 'a) => 'b ) => array ('b)
```

Reply:
The dot represents a `[@bs]` payload. Here the `[@bs]` sits on the `('a => 'b)`
```reason
external map : ([@bs] ('a => 'b)) => array('b) = "";
external map : (. ('a => 'b)) => array('b) = "";
```